spring-boot/admin/src/main/java/eu/coatrack/admin/model/repository/MetricsAggregationCustomRepository.java

```diff
@@ -379,6 +376,38 @@ public long getNumberOfApiCallers(LocalDate from, LocalDate until, String apiPro
         return getSummarizedMetrics(serviceId, null, fromDate, untilDate);
     }
 
+    public List<StatisticsPerService> getStatisticsOfServicesOwnedByLoggedInUserAndByDateRange(
+            List<ServiceApi> servicesOwnedByLoggedInUser,
+            LocalDate fromDate,
+            LocalDate untilDate) {
+
+        List<StatisticsPerService> listOfStatisticsPerServiceOwnedByLoggedInUser = new ArrayList<>();
+        return addServiceStatisticToListOfStatistics(listOfStatisticsPerServiceOwnedByLoggedInUser,
+                servicesOwnedByLoggedInUser, fromDate, untilDate);
+    }
+
+    public List<StatisticsPerService> addServiceStatisticToListOfStatistics(List<StatisticsPerService> listOfStatisticsPerServiceOwnedByLoggedInUser,
+                                                                            List<ServiceApi> servicesOwnedByLoggedInUser, LocalDate fromDate,
+                                                                            LocalDate untilDate) {
+        servicesOwnedByLoggedInUser.forEach((service) -> {
```

Comment:
The brackets around `service` are redundant.

Reply:
Done